api/v2/Routes/students/index.js

```diff
@@ -22,4 +24,14 @@ router.use('/:id/grades', GradesRouter);
 router.use('/:email/projections', ProjectionsRouter);
 router.use('/:id/progressquerystring', ProgressQueryStringRouter);
 
+router.get('/', validateAdminMiddleware, async (_, res) => {
+    try {
+        const students = await getStudents();
+        return res.status(200).json({ students });
+    } catch (error) {
+        console.error('Error retrieving students from Redis:', error);
+        return res.status(500).json({ message: 'Internal Server Error' });
+    }
```

Comment:
These errors should already be handled by our default error handler.  Thats the only place a 500 should be thrown.